pages/api/id/[form]/submission/report.ts

```diff
@@ -278,8 +278,10 @@ async function notifySupport(
     language,
   });
 
-  if (result && result?.status >= 400) {
-    throw new Error(`Freshdesk error: ${result.status}`);
+  if (result.status >= 400) {
```

Comment:
It would be beneficial to create a variable named "BAD_REQUEST" for the HTTP status code 400. In the case of a bad request, should we log the request?

Reply:
According to https://support.freshdesk.com/en/support/solutions/articles/225440-possible-error-codes-and-its-reasons

Any errors should be 400+

These are already logging but we're not getting enough info 
<img width="345" alt="Screenshot 2024-01-25 at 9 52 16 AM" src="https://github.com/cds-snc/platform-forms-client/assets/62242/cb74c79a-9d5d-43b0-a889-75bd795d5dd0">